Fix to add support for the DESTDIR enviroment variable

Index: uqm-hd-0.1/build.vars.in
===================================================================
--- uqm-hd-0.1.orig/build.vars.in	2012-12-31 10:40:34.366742407 -0500
+++ uqm-hd-0.1/build.vars.in	2012-12-31 10:40:37.066755804 -0500
@@ -31,9 +31,9 @@
 WINDRES='@WINDRES@'
 uqm_CFLAGS='@CFLAGS@'
 uqm_LDFLAGS='@LDFLAGS@'
-uqm_INSTALL_BINDIR='@INSTALL_BINDIR@'
-uqm_INSTALL_LIBDIR='@INSTALL_LIBDIR@'
-uqm_INSTALL_SHAREDIR='@INSTALL_SHAREDIR@'
+uqm_INSTALL_BINDIR=$DESTDIR'@INSTALL_BINDIR@'
+uqm_INSTALL_LIBDIR=$DESTDIR'@INSTALL_LIBDIR@'
+uqm_INSTALL_SHAREDIR=$DESTDIR'@INSTALL_SHAREDIR@'
 
 # Exported variables are available from all the Makeinfo files
 # Non-exported files only where build.vars is explicitely included.
