--- a/src/svnversion.h
+++ /dev/null
@@ -1 +0,0 @@
-#define UQMHD_SVN_REVISION "894:895M"
\ No newline at end of file
--- a/src/uqm/restart.c
+++ b/src/uqm/restart.c
@@ -41,7 +41,7 @@
 #include "util.h"
 #include "starcon.h"
 #include "uqmversion.h"
-#include "svnversion.h"
+//#include "svnversion.h"
 #include "libs/graphics/gfx_common.h"
 #include "libs/inplib.h"
 
@@ -71,7 +71,7 @@
 	STAMP s;
 	TEXT t;
 	UNICODE buf[64];
-	COUNT svn_revision = 0; // JMS
+//	COUNT svn_revision = 0; // JMS
 
 	//DC: Load the different menus depending on the resolution factor
 	if (resolutionFactor < 1)
@@ -106,6 +106,8 @@
 	LockMutex (GraphicsLock);
 	DrawStamp (&s);
 
+/* SVN revision printouts are not needed in the Debian package (and don't even work),
+ * so we remove them.
 	// JMS: Hack for printing out the correct SVN revision number.
 	{
 		UNICODE svn_buf[64] = {0};
@@ -132,7 +134,7 @@
 		// Think about this for a while ;)
 		++svn_revision;
 	}
-	
+*/	
 	// Put the version number in the bottom right corner.
 	SetContextFont (TinyFont);
 	t.pStr = buf;
@@ -140,14 +142,17 @@
 	t.baseline.y = SCREEN_HEIGHT - 2;
 	t.align = ALIGN_RIGHT;
 	t.CharCount = (COUNT)~0;
-	sprintf (buf, "v%d.%d.%d%s - SVN r%d", UQM_MAJOR_VERSION, UQM_MINOR_VERSION, UQM_PATCH_VERSION, UQM_EXTRA_VERSION, svn_revision);
+//	sprintf (buf, "v%d.%d.%d%s - SVN r%d", UQM_MAJOR_VERSION, UQM_MINOR_VERSION, UQM_PATCH_VERSION, UQM_EXTRA_VERSION, svn_revision);
+	sprintf (buf, "v%d.%d.%d%s", UQM_MAJOR_VERSION, UQM_MINOR_VERSION, UQM_PATCH_VERSION, UQM_EXTRA_VERSION);
 	SetContextForeGroundColor (WHITE_COLOR);
 	font_DrawText (&t);
 
 	UnlockMutex (GraphicsLock);
 	UnbatchGraphics ();
+
 }
 
+
 // JMS_GFX: The cleanup boolean can be used to avoid drawing a wrong-sized "Setup" flash overlay.
 static void
 DrawRestartMenu (MENU_STATE *pMS, BYTE NewState, FRAME f, BOOLEAN cleanup)
--- a/build/unix/build_functions
+++ b/build/unix/build_functions
@@ -160,9 +160,6 @@
 	. "${BUILD_REC_PATH:=./}Makeproject"
 
 	
-	printf "%s" "#define UQMHD_SVN_REVISION \"" > src/svnversion.h
-	svnversion -n . >> src/svnversion.h
-	printf "%s" "\"" >> src/svnversion.h
 }
 
 # Description: check if the necessary depend file is present,
