--- uqm-improved-netmelee-2.06.orig/Makeproject
+++ uqm-improved-netmelee-2.06/Makeproject
@@ -25,7 +25,7 @@ fi
 # config.
 uqm_INSTALL_LIBS="executable"
 uqm_INSTALL_LIB_executable_SRC="$BUILD_WORK/$uqm_NAME"
-uqm_INSTALL_LIB_executable_DEST=uqm/uqm
+uqm_INSTALL_LIB_executable_DEST=uqm/$uqm_NAME
 uqm_INSTALL_LIB_executable_MODE="0755"
 
 # Stuff to install under the directory for system-independant data, as
