Fix to add support for the DESTDIR enviroment variable

Index: uqm-0.7.0/build.vars.in
===================================================================
--- uqm-0.7.0.orig/build.vars.in	2012-08-30 14:20:49.000000000 -0400
+++ uqm-0.7.0/build.vars.in	2012-08-30 14:21:54.000000000 -0400
@@ -31,9 +31,9 @@
 WINDRES='@WINDRES@'
 uqm_CFLAGS='@CFLAGS@'
 uqm_LDFLAGS='@LDFLAGS@'
-uqm_INSTALL_BINDIR='@INSTALL_BINDIR@'
-uqm_INSTALL_LIBDIR='@INSTALL_LIBDIR@'
-uqm_INSTALL_SHAREDIR='@INSTALL_SHAREDIR@'
+uqm_INSTALL_BINDIR="$DESTDIR"'@INSTALL_BINDIR@'
+uqm_INSTALL_LIBDIR="$DESTDIR"'@INSTALL_LIBDIR@'
+uqm_INSTALL_SHAREDIR="$DESTDIR"'@INSTALL_SHAREDIR@'
 
 # Exported variables are available from all the Makeinfo files
 # Non-exported files only where build.vars is explicitely included.
